test/Tester/GrainServiceTests/CustomGrainService.cs

```diff
@@ -33,6 +47,22 @@ public CustomGrainService(IGrainIdentity id, Silo silo, IGrainServiceConfigurati
             
         }
 
+        private bool m_Started = false;
```

Comment:
nit: the convention used in this codebase it to name private fields in lower-camelcase with no prefix